source/main/index.js

```diff
@@ -103,37 +64,17 @@ const onAppReady = async () => {
     client.create(mainWindow);
   }
 
-  // Build app menus
-  let menu;
-  if (isMacOS) {
-    menu = Menu.buildFromTemplate(osxMenu(app, mainWindow, menuActions, isInSafeMode));
-    Menu.setApplicationMenu(menu);
-  } else {
-    menu = Menu.buildFromTemplate(winLinuxMenu(app, mainWindow, menuActions, isInSafeMode));
-    mainWindow.setMenu(menu);
-  }
-
-  // Hide application window on Cmd+H hotkey (OSX only!)
-  if (isMacOS) {
-    app.on('activate', () => {
-      if (!mainWindow.isVisible()) app.show();
-    });
-
-    mainWindow.on('focus', () => {
-      globalShortcut.register('CommandOrControl+H', app.hide);
-    });
-
-    mainWindow.on('blur', () => {
-      globalShortcut.unregister('CommandOrControl+H');
-    });
-  }
-
   mainWindow.on('close', async (event) => {
     Logger.info('mainWindow received <close> event. Safe exiting Daedalus now.');
     event.preventDefault();
     await safeExit();
   });
 
+  // TODO: Figure out how to get the locale
+  // const locale = 'en-US.json';
+  const locale = 'ja-JP.json';
```

Comment:
@daniloprates you should use electron-storage to read this info: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/api/utils/localStorage.js#L30
You can use this call in the main process to. Just be aware that initially before user selects the language from the initial setup language-selection we should always fallback to English. That exactly what we do in the rendering process: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/stores/ProfileStore.js#L99-L103

Reply:
@MarcusHurney FYI - read my previous comment 🙏